app/src/main/java/ar/com/wolox/android/example/ui/news/NewsFragment.kt

```diff
@@ -67,7 +66,16 @@ class NewsFragment @Inject constructor() : WolmoFragment<NewsPresenter>(), NewsV
         newsAdapter!!.addNews(news)
     }
 
+    override fun onNewClick(new: New) {
+        NewDetailActivity.start(requireContext(), new, context?.getSharedPreferences("ar.com.wolox.android.example.ui.login", Context.MODE_PRIVATE)!!.getInt("user_id", 0))
```

Comment:
If shared preferences are injected , I don't understand the third parameter. Moreover, using file path as keys is not a good practice